React/Base/RCTConvert.h

```diff
@@ -92,7 +92,7 @@ typedef NSURL RCTFileURL;
 + (UIFont *)UIFont:(UIFont *)font withStyle:(id)json;
 + (UIFont *)UIFont:(UIFont *)font withFamily:(id)json;
 + (UIFont *)UIFont:(UIFont *)font withFamily:(id)family
-              size:(id)size weight:(id)weight style:(id)style
+              size:(id)size weight:(id)weight style:(id)style variant:(id)variant
```

Comment:
I wonder if we can make all of these params typed instead of using `id` everywhere.